### ISSUE TITLE
PFObject manipulation causes incorrect results locally

### ISSUE THREAD

**Issue by acegreen** [parse-community/Parse-SDK-iOS-OSX#694]
So take an object with two columns "Shorted_By" and "Longed_By"

Initial Status:
"Shorted_By" = []
"Longed_By" = [PFUser1]

Then I do the following to the object:

```
        currentObject!.removeObject(PFUser.currentUser()!, forKey: "Shorted_By")

        if currentObject!.objectForKey("Longed_By") != nil {

            currentObject!.addUniqueObject(PFUser.currentUser()!, forKey: "Longed_By")

        } else {

            currentObject.setObject([PFUser.currentUser()!], forKey: "Longed_By")
        }
```

In the case above the currentUser happens to be PFUser2. 

The outcome, when I print object (BOTH before saving OR after): INCORRECT result
"Shorted_By" = [PFUser2]
"Longed_By" = [PFUser1, PFUser2]

In Database: CORRECT result
"Shorted_By" = []
"Longed_By" = [PFUser1, PFUser2]

Basically it saves it correctly in the database but the object at hand gets corrupted. This implies even if both columns were empty, it adds PFUser2 to both but shows correctly in database afterwards.

Why is that? 

<img width="689" alt="screen shot 2015-12-25 at 10 21 50 pm" src="https://cloud.githubusercontent.com/assets/10794609/12005107/d67c1caa-ab5d-11e5-8511-734a06e64674.png">
<img width="1287" alt="screen shot 2015-12-25 at 10 22 58 pm" src="https://cloud.githubusercontent.com/assets/10794609/12005108/d68a9834-ab5d-11e5-9474-275a329c6269.png">

**Comment by nlutsenko** [comment]
Super strange and off-the-top of my head - it shouldn't happen.
Any chance you can add a bigger code snippet/sample project that pinpoints the issue?

**Comment by acegreen** [comment]
@nlutsenko we already had this topic before, I won't send my project nor will I sit down and recreate another one. I don't get paid by Parse, but you do. I fetched and object, did the above to it and printed it out. The results are as above. 

As I said before, you can take that or close this problem until someone else finds it. Sorry for the bluntness but I feel thats your go to for everything "create a project for me"
I offered a Skype chat to show you my localDatabase store issue but didn't like that until the ticket was closed.

**Comment by acegreen** [comment]
@nlutsenko Does 1.12.0 address this?

**Comment by acegreen** [comment]
??

**Comment by Tom-Fox1** [comment]
Closing this as no one has reported this issue recently, and we don't have the resources to investigate these historic reports.

Please ask me to reopen if you experience this issue in the latest version of this SDK.
